frontstage_api/resources/register/verify_email.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from flask import request
+from flask_restplus import Resource
+from structlog import wrap_logger
+
+from frontstage_api import api
+from frontstage_api.controllers import party_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+@api.route('/verify-email')
+class VerifyEmail(Resource):
+
```

Comment:
Same as above

Reply:
In general we need to agree on how we're documenting our code across RAS, a lot of services have very little/none/different formats.

For frontstage API I'm going to add extra validation onto the endpoints with marshmallow (comes with flask-restplus). There's also really good swagger documentation you can add easily but I was going to implement all that in a separate PR

Reply:
I don't think we're using swagger now, I was asked to update the secure messaging to contain an API.md file with all the endpoints in.

Reply:
we're not using swagger atm, flask-restplus has build in functionality to generate swagger documentation very easily, I have a branch which does it already for some of the endpoints.

We need to agree on what we're doing across RAS/RM though so we can be consistent

Reply:
We should have a discussion on the way we should be doing api documentation. Ideally documentation should live as close to the code as possible. This is what swagger does but it can be implemented badly in some cases. We should have a discussion @pricem14pc @davidmort?

Reply:
happy to have a discussion around this, i believe we used to use swagger (its what i've used for api docs in the past) but have moved away from it for some reason so just need to get some understanding around that, in the short term stick to current "API.md" method

Reply:
I believe the API.md is to be consistent across the whole of SDC